### DISCUSSION TITLE
Scroll Bars

### DISCUSSION THREAD

**Discussion by lucasaarcoverde** [vtex/shoreline#826] [Design RFCs]
Currently, scrollbars are not customized. There's a CSS-only solution to override the browser's default scrollbar that works well across browsers without the overhead and complications of other custom scroll bars solutions on the lines of [simplebar-react](https://github.com/Grsmto/simplebar/tree/master/packages/simplebar-react). Even so, we have [discussed this issue and decided that it's not worth it to customize scroll bars](https://vtex.slack.com/archives/C02NJHMAY10/p1649966625162899).

We need to document our approach to scroll bars somewhere, and the rationale behind it. We also need to review all components that currently use a custom scroll bar (only the admin shell?) to remove the customization.

**Comment by lucasaarcoverde** [comment]
@davicostalf Does this still makes sense? It seems not scalable to me

**Comment by marcelovicentegc** [comment]
I'm with Lucas ☝️

**Comment by davicostalf** [comment]
Updated the description to reflect what is the issue that is pending

**Comment by marcelovicentegc** [comment]
@davicostalf, there are no custom scroll bars in use 😉

**Comment by matheusps** [comment]
We will not customize scrollbars, it is still not reliable/scalable (jan 2024)
